webui/react/src/components/InlineEditor.module.scss

```diff
@@ -111,3 +111,26 @@
 .base.disabled .growWrap:hover > textarea {
   border-color: transparent;
 }
+.shakeAnimation {
+  animation: shake 0.8s cubic-bezier(0.36, 0.07, 0.2, 0.95) both;
```

Comment:
**question:** would it be possible (and do you think it'll be a good idea) to change the border to `var(--theme-status-critical)` as it's shaking and back to the regular border color to show that it's a problem?

Reply:
I think its possible and a good idea. My concern is that if the entered value is invalid, the previous value is set immediately. Ideally setting the previous value after shaking would be better with the `critical border`. Let me try

Reply:
I fixed it. It was actually not hard to do

https://user-images.githubusercontent.com/109113616/183133976-92a3f6bc-c25d-485d-b5e1-c519040ba77c.mov